staging/kos/Makefile

```diff
@@ -46,11 +47,73 @@ deploy/main/50-rs-ipam.yaml: deploy.m4/main/50-rs-ipam.yaml.m4
 deploy/main/50-ds-ca.yaml: deploy.m4/main/50-ds-ca.yaml.m4
 	m4 -DDOCKER_PREFIX=${DOCKER_PREFIX} deploy.m4/main/50-ds-ca.yaml.m4 > deploy/main/50-ds-ca.yaml
 
+etcd-tls/certs/ca.pem:
+	rm -rf etcd-tls/certs
+	mkdir etcd-tls/certs
+	openssl genrsa -out etcd-tls/certs/ca-key.pem 4096
+	openssl req -x509 -new -nodes -key etcd-tls/certs/ca-key.pem -days 10000 \
```

Comment:
Why `ca-key.pem` but `peer.key`, `server.key`, and `etcd-client.key`?

Reply:
I looked at some examples and apparently it's common to create keys with trailing .pem, so I though it was desirable. The etcd operator however expects `peer.key`, `server.key`, and `etcd-client.key` with those exact names (and creating them with .pem and renaming them seemed to add clutter).

Reply:
Ugh.  Changing `ca-key.pem` to `ca.key` would make it more regular.